ccd-definition/CaseEvent/User/UserEvents-SDO-nonprod.json

```diff
@@ -16,8 +16,7 @@
     "EndButtonLabel": "Submit",
     "RetriesTimeoutAboutToStartEvent": 0,
     "RetriesTimeoutURLAboutToSubmitEvent": 0,
-    "Publish": "Y",
```

Comment:
For testing?

Reply:
I would assume so. This change came from the base branch https://github.com/hmcts/civil-ccd-definition/blob/hearingFeePaidJourney/ccd-definition/CaseEvent/User/UserEvents-SDO-nonprod.json
I used Max's branch because I needed his code, but my PR is not back into his branch but into master, that's why it shows all differences from master, mine and his. It's a bit inconvenient, but the assumption was that his branch would be merge into master earlier that mine, since my ticket is dependent on other ticket that is not in development yet.

Reply:
Yes, this is for testing only. I'll gate it so it doesn't get merged without this being resolved. Otherwise the branch looks fine.

Reply:
Gated.